pkg/api/types.go

```diff
@@ -673,6 +673,9 @@ type GlusterfsVolumeSource struct {
 	// Required: EndpointsName is the endpoint name that details Glusterfs topology
 	EndpointsName string `json:"endpoints"`
 
+	// Optional: Servers is the gluster server list as a string seperated by a comma.
+	Servers string `json:"servers,omitempty"`
```

Comment:
comma separate list is not common in our APIs.   Usually one sees `[]string`.  Is comma separate list the standard for specifying Glusterfs server lists?